runtime/kusama/src/xcm_config.rs

```diff
@@ -157,7 +157,7 @@ pub type Barrier = (
 pub struct SafeCallFilter;
 impl Contains<RuntimeCall> for SafeCallFilter {
 	fn contains(call: &RuntimeCall) -> bool {
-		#[cfg(feature = "runtime-benchmarks")]
+		#[cfg(any(feature = "std", feature = "runtime-benchmarks", test))]
```

Comment:
dumb question: does enabling it with `std` feature means that native execution will differ from wasm?

Reply:
Ahh yeah. Good point. I have overseen this :see_no_evil: